man/man8/zfsprops.8

```diff
@@ -1109,7 +1109,10 @@ to access the encrypted data (see
 for details). This setting will also allow the key to be passed in via STDIN,
 but users should be careful not to place keys which should be kept secret on
 the command line. If a file URI is selected, the key will be loaded from the
-specified absolute file path.
+specified absolute file path. If an exec URI is selected, the key will be
+acquired from the executable specified (see EXEC:// in
+.Xr zfs-load-key 8 )
+for details.
```

Comment:
From reading the two man pages, it's not immediately clear to me, unless I also read the code examples, that this command receives actions via arguments passed to it. I think the limitation that this can only take a single executable and not additional flags for the command should be documented.

Reply:
Granted, it's at the end of that list, but at the beginning of the sexion, I'm not sure how I'd make it clearer:
> The provider is invoked with its full path as its zeroeth argument, the operation (lower-case) as its first argument, and the dataset name, if any, as its third argument.

I mean, it's not like a path *can* contain flags, is it?